src/main/java/com/minecolonies/coremod/colony/permissions/ForgePermissionNodes.java

```diff
@@ -0,0 +1,74 @@
+package com.minecolonies.coremod.colony.permissions;
+
+import net.minecraftforge.server.permission.DefaultPermissionLevel;
+
+/**
+ * Available forge Permission nodes that can be checked for access.
+ *
+ * See this url until real documentation becomes available.
+ * https://github.com/MinecraftForge/MinecraftForge/pull/3155
+ *
+ */
+public enum ForgePermissionNodes
+{
+    RandomTeleport ("com.minecolonies.coremod.RandomTeleport", DefaultPermissionLevel.OP, "Can randomly teleport...."),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00115)